README.md

```diff
@@ -23,92 +24,100 @@ The main OpenStack entry point is at http://www.openstack.org
+ * P(icture) service: Look into Swift or Redis with customer ID to display the image of the customer, empty if no image.
+ * Redis or Swift can be used to store data.
+ * Rabbitmq is used to pass message from service B to service W1 and W2.
+ * W1 services that listen on the messaging bus and write to the database.
+ * W2 services that listen on the messaging bus and send http requests to mailgun external services.
 
- 1. The DB info should resist to failure and not loose records (Using a message bus e.g. to handle failures).
- 2. The application provided has some latency. Use OpenStack to scale it up (message bus/ha proxy) in order to be able to exceed the 4 reqs on a lapse of 10 seconds. A LBaaS (Load Balancer as a Service) can also be used and provided as part of the solution.
- 3. Monitoring, Security, Error case, Full HA, Full install/conf automation, Automatic Scalability.
+## Application materials (code, doc etc...)
+https://github.com/uggla/cloud_native_app
 
-This part will be **evaluated on 4 additional bonus points**.
 
 ## Deliverables
 
- * On Gihub: 
-   * Heat template/ansible playbooks/scripts for Infra group
-   * Performance and tests results
-   * Application code
- * Application design document
- * Present the (almost !) automatic deployment of the application in a new empty tenant and make reliability checks.
+ * On Gihub:
+   * Pipeline design documents.
+   * Application code.
+   * Heat template/ansible playbooks/scripts for Infra group.
+
+ * Video that will present the CICD of the application in a new empty tenant.
 
 ## Teachers support
 
-Teachers will provide a remote access to the pre-production platform (set of servers preinstalled with a CentOS 7 distribution) that the IaaS team will have to setup from an OpenStack perspective and deliver so the DevOps teams could deploy their application on it. Teachers will also provide a production platform running an OpenStack distribution as a development environment for the application developers for testing purposes. 
-A mailing list has been created to provide remote support to students during the whole period. Various people will so be available to answer questions and help with regards to platform setup and access. The mailing-list address is ensimag-openstack@lists.osp.hpe.com 
+Teachers will provide a remote access to the production platform (set of servers preinstalled with a CentOS 7 distribution) that the team will have to setup from an OpenStack perspective and deliver teams could deploy their application on it. Teachers will also provide a pre-production platform running an OpenStack distribution as a development environment for the application developers for testing purposes.
```

Comment:
I thought we were calling it reference platform not pre-prod ;-)))

Reply:
Correct. Look how I named it in my proposal... Trying to bring it in the story...